scripts/run_docker_build.sh

```diff
@@ -50,10 +51,10 @@ conda install conda-forge-build-setup
 source run_conda_forge_build_setup
 
 # yum installs anything from a "yum_requirements.txt" file that isn't a blank line or comment.
-find conda-recipes -mindepth 2 -maxdepth 2 -type f -name "yum_requirements.txt" \
+find ~/conda-recipes -mindepth 2 -maxdepth 2 -type f -name "yum_requirements.txt" \
     | xargs -n1 cat | grep -v -e "^#" -e "^$" | \
-    xargs -r yum install -y
+    xargs -r /usr/bin/sudo -n yum install -y
```

Comment:
I think it is not necessary.
I confused with the sudoers content because if you do ```echo 'conda ALL=NOPASSWD: yum' >> /etc/sudoers``` it does not work